apps/api/src/services/form.service.ts

```diff
@@ -4,25 +4,76 @@ import {
 	NotFoundException,
 } from '@nestjs/common';
 import { PrismaService } from './prisma.service';
-import { AiService } from './ai.service';
 import { EventsService } from './events.service';
 import { FormFieldsModel, FormModel } from '@api/models';
-import { CreateFormFieldDto } from '@api/dto/form-field.dto';
+import { CreateFormFieldDto, EditFormFieldDto } from '@api/dto/form-field.dto';
 import { SystemTable } from '@api/utils/db';
 import { FFError } from '@api/utils/error';
+import { NewFormDto } from '@api/dto/form.dto';
 
 @Injectable()
 export class FormService {
 	constructor(
 		private readonly prismaService: PrismaService,
-		private readonly AiService: AiService,
 		private readonly EventService: EventsService,
 	) {}
 
 	async getAllForm(eventId: string) {
 		return await FormModel.getAllFormsWithSubmissionsCount(eventId);
 	}
 
+	async createNewForm(data: NewFormDto, eventId: string) {
+		await FormModel.insert({
+			title: data.title,
+			description: data.description,
+			fk_event_id: eventId,
+			confirmation_message: '',
+			misc: {},
```

Comment:
@DarkPhoenix2704  what is misc in form. ?

Reply:
For some future use cases. Let's keep it null, if not used

as said earlier, don't desctructure and add like this. Instead use


```ts
await FormModel.insert(
    ...data,
    fk_event_id: eventId
)

```